### PR TITLE
finished task

### PR DESCRIPTION
**Pull request by trtskvalerie**
- [DEMO LINK](https://trtskvalerie.github.io/layout_search-bar-airbnb/src)
- [TEST REPORT LINK](https://trtskvalerie.github.io/layout_search-bar-airbnb/backstop_data/html_report/index.html)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf");
+}
+
+.search-bar {
+  width: var(--bar-w);
```
Comment (Anna-Tkachenko) [inline src/style.css:17]: div has 100% width by default

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,48 @@
...
+
+    <form action="http://localhost:8080/api">
+      <div class="search-bar search-bar-sm">
+
+        <div class="loop-container-sm">
+          <img
+            src="images/Search.svg"
+            alt="loop"
+            class="loop-sm"
+          >
```
Comment (Anna-Tkachenko) [inline src/index.html:44]: You can also add image as an input background or use ::before

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:8080/api">
```
[the remark this answers is not in the document]
Comment (Anna-Tkachenko) [reply src/index.html:14]: It's better to add a method attribute for form

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  height: var(--height-sm);
+}
+
+.search-field {
+  float: right;
+  padding: 0;
+  border: 0;
+  border-radius: 5px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
```
Comment (Anna-Tkachenko) [inline src/style.css:55]: It's better to set default style to html tag and use font-family: inherit for inputs

**Review by Anna-Tkachenko — APPROVED** [review]

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**